### ISSUE TITLE
Syntax coloring bug 

### ISSUE THREAD

**Issue by XsongyangX** [microsoft/pylance-release#591]
The syntax coloring for the Python language in VSCode has a subtle bug. The inner closing parenthesis has the same color as its elements, which is wrong. See the attached screenshot for reference. The bug is right after `finished_queuing`.

## Environment data

Version: 1.51.0 (user setup)
Commit: fcac248b077b55bae4ba5bab613fd6e9156c2f0c
Date: 2020-11-05T18:18:23.642Z
Electron: 9.3.3
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Windows_NT x64 10.0.19041
-   Python version (& distribution if applicable, e.g. Anaconda): 3.8.5 64-bit (conda)
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Value of the `python.languageServer` setting: Pylance

[**NOTE**: If you suspect that your issue is related to the Microsoft Python Language Server (`python.languageServer: 'Microsoft'`), please download our new language server [Pylance](https://marketplace.visualstudio.com/items?itemName=ms-python.vscode-pylance) from the VS Code marketplace to see if that fixes your issue]

## Steps to reproduce:

[**NOTE**: Self-contained, minimal reproducing code samples are **extremely** helpful and will expedite addressing your issue]

1. Open a Python scripts and type the following
```python
finished_queuing = Event()
threading.Thread(target=labels_dumper, args=(finished_queuing)).start()
return finished_queuing
```
2. You'll see a syntax discoloring on the closing parenthesis for args=

## Logs
![image](https://user-images.githubusercontent.com/10444460/98586745-291b6780-2297-11eb-98ec-f6239c2467a8.png)

**Comment by jakebailey** [comment]
This appears to have changed since being reported, now the both parens are a part of the arg:

![image](https://user-images.githubusercontent.com/5341706/106978478-35398880-6711-11eb-9750-4b7d7ec83a56.png)

**Comment by XsongyangX** [comment]
Cool! I'll close this issue. If the problem comes back, feel free to re-open it! 👍

**Comment by jakebailey** [comment]
I don't think the bug is gone, just changed. I don't believe we should be highlighting parens like this... Note how we don't mark the other parens as blue when they are in a call expression, for example.

**Comment by cdce8p** [comment]
This seems to also apply to type annotations.

<img width="466" alt="Screen Shot 2021-03-12 at 12 02 43" src="https://user-images.githubusercontent.com/30130371/110931703-dcba6400-832a-11eb-8e94-bf2c3d9e0aad.png">

I'm aware that the parentheses aren't necessary here. Just though it might be helpful to know.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.9.2, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202192-16-september-2021
